### PR TITLE
[DOCS] Add info about location to register Slots

### PR DESCRIPTION
**Pull request by sbusemann**

### CODE REVIEW

**Comment by sbusemann** [comment]
As an extension developer, I do not know, where to register Slots

**Comment by DavidBruchmann** [comment]
Sorry, that doesn't exist anymore.

image

**Comment by sbusemann** [comment]
TYPO3 11 will be in use for quite some years

File: Documentation/ApiOverview/Events/SignalsSlots/Index.rst [hunk 1]
```diff
@@ -121,6 +121,8 @@ and take a look at the dispatch call:
 In this case, the :php:`dumpClassLoadingInformation` method will get the
 extension key and an instance of the dispatching class as parameters.
 
+If you are using Signals in your custom extension, you need to register the Slot in your ext_localconf.php.
```
Comment (brotkrueml) [inline Documentation/ApiOverview/Events/SignalsSlots/Index.rst:124]:
```suggestion
If you are using Signals in your custom extension, you need to register the Slot in your :file:`ext_localconf.php`.
```

**Review by brotkrueml — APPROVED** [review]

**Comment by brotkrueml** [comment]
@sbusemann Thanks for your contribution :+1:

**Comment by DavidBruchmann** [comment]
AN example how to use SignalSlotDispatcher can be found here:
https://docs.typo3.org/m/typo3/reference-coreapi/6.2/en-us/Internationalization/Translation/Index.html?highlight=signalslot#custom-translation-servers

**Pull request merged**